### DISCUSSION TITLE
Error with EdgeX UI (Minnesota) in secure mode accessing App Service tab

### DISCUSSION THREAD

**Discussion by pachori91** [edgexfoundry/community#224] [General]
Hi Team, 

i just started edgex (in secured mod) by using docker-compose from [HERE](https://github.com/edgexfoundry/edgex-compose/blob/minnesota/docker-compose.yml).

All services running fine but while i accessed Edgex UI tried with Tab "**AppService**" then i getting error as below-

`**Svc Unavailable
Svc Unavailable
Oops!!! registry center service is unavailable!
This could be a network reason, or it could be that the registry center microservice is not running, make sure there is no network problem or that the registry center service is already running!

EdgeXFoundry uses a microservices architecture, each service is running independently, please make sure that the current service is already running, if you have manually started the service, click the Refresh button**.`

Please suggest me how can i resolve this issue.

**Comment by lenny-goodell** [comment]
@pachori91, This is a known issue with the current UI. There isn't a work around.
